docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -134,6 +102,37 @@ If you want to provision a new volume or restore the existing one, create on-dem
       apiGroup: snapshot.storage.k8s.io
   ```
 
+  </details>
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.17.
+
+### Steps
+
+  1. [Install the CSI driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md).
+  2. [Create a volume snapshot](https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/deploy/example/snapshot).
```

Comment:
I would rather recommend we use the same steps mentioned here for gardener. So 
1. install CSI Driver
2. Create snapshot as mentioned in steps here: https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/10-backup-kyma#back-up-kyma--backup-providers--gardener-gcp--steps for azure

Reply:
Wait, which instructions are the good example, and which ones should be changed?

Should I change the Gardener instructions so they follow the AKS (?) example? Or based on the Azure example - which is already part of the Gardener instructions? Or should I adjust the AKS instructions?